invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoansSearch/LoansSearch.js

```diff
@@ -47,43 +46,63 @@ export class LoansSearch extends Component {
     );
   };
 
-  prepareData(data) {
-    return data.map(row => {
-      const actions = row.metadata.is_overdue && (
-        <OverdueLoanSendMailModal loan={row} />
-      );
-      return _pick(formatter.loan.toTable(row, actions), [
-        'ID',
-        'Document ID',
-        'Patron',
-        'State',
-        'Request start date',
-        'End date',
-        'Renewals',
-        'Actions',
-      ]);
-    });
-  }
+  viewDetails = ({ row }) => {
+    return (
+      <Button
+        as={Link}
+        to={BackOfficeRoutes.loanDetailsFor(row.metadata.pid)}
+        compact
+        icon="info"
+        data-test={row.metadata.pid}
+      />
+    );
+  };
 
   renderResultsTable = results => {
-    const rows = this.prepareData(results);
     const maxRowsToShow =
-      rows.length > ResultsTable.defaultProps.showMaxRows
-        ? rows.length
+      results.length > ResultsTable.defaultProps.showMaxRows
+        ? results.length
         : ResultsTable.defaultProps.showMaxRows;
 
     const headerActionComponent = (
       <ExportReactSearchKitResults exportBaseUrl={loanApi.searchBaseURL} />
     );
 
+    const columns = [
+      { title: '', field: '', formatter: this.viewDetails },
+      { title: 'ID', field: 'metadata.pid' },
+      { title: 'Document ID', field: 'metadata.document_pid' },
+      { title: 'Patron', field: 'metadata.patron.name' },
```

Comment:
I am not sure about links in the search results, but maybe same here too...

Reply:
If I understood correctly, you can check it in action here, https://books-dev.web.cern.ch/backoffice/stats the amount of total loans is the actual query for them